.github/ISSUE_TEMPLATE/core-chore-template.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Core Chore Template
+about: Use this template for Notify Core chores
+title: "[CHORE] "
+labels: Chore, Routine, Tech Debt
```

Comment:
I think these would refer to existing labels we have. We could create Routine or Chore, I would just keep one of these and not both.

Reply:
Ok I see we have both existing, but Chore refers to no issue and Routine does. I would like to delete Chore then, what do you think?

Reply:
```suggestion
labels: Routine, Tech Debt
```

Reply:
I created chore not seeing routine - you're right though. I'll delete Chore.